api/_examples/datadog-alert-channel/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func main() {
+	lacework, err := api.NewClient("account", api.WithApiKeys("KEY", "SECRET"))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	myDatadogChannel := api.NewDatadogAlertChannel("datadog-alert-from-golang",
+		api.DatadogChannelData{
+			DatadogSite:    "eu",
+			DatadogService: "Events Summary",
```

Comment:
I don't think this code would work as is...
```suggestion
			DatadogSite:    api. DatadogSiteEu,
			DatadogService: api. DatadogServiceEventsSummary,
```